tools/admin/batch_invite.py

```diff
@@ -0,0 +1,36 @@
+from github import Github
+import os
+
+token = os.environ.get("TOKEN")
+organization_name = os.environ.get("ORG_NAME")
```

Comment:
There's no `gh` equivalent, but still putting it under `GH_` seems nice

Reply:
Fixed.